pt/lessons/ecto/associations.md

```diff
@@ -9,15 +9,15 @@ Nessa seção vamos aprender a utilizar o Ecto para definir e trabalhar com asso
 
 ## Configuração
 
-Nós vamos construir a aplicação `Friends`, das últimas lições. Você pode referir-se a configuração [aqui](https://elixirschool.com/pt/lessons/ecto/basics) para uma breve recapitulação.
+Nós vamos começar com a mesma aplicação `Friends`, das últimas lições. Você pode referir-se a configuração [aqui](https://elixirschool.com/pt/lessons/ecto/basics) para uma breve recapitulação.
```

Comment:
so...

_Nós vamos começar com a mesma aplicação `Friends`, das últimas lições._

I think about this sentence and try to think in a sequential flow and in my opinion, it could be a little bit better if we say something like:

_Nós vamos utilizar a mesma aplicação `Friends`_  das últimas lições.

what do you think about it?